xml/templates/message_templates/participant_confirm_html.tpl

```diff
@@ -23,22 +23,22 @@
     <p>{ts}This is an invitation to complete your registration that was initially waitlisted.{/ts}</p>
    </td>
   </tr>
-  {if !$isAdditional and $participant.id}
+  {if !$isAdditional and {participant.id|boolean}}
```

Comment:
Something about this isn't working. It's false so there's no links in the email!